hass_nabucasa/remote.py

```diff
@@ -143,9 +148,16 @@ def certificate(self) -> Optional[Certificate]:
             try:
                 await self._acme.issue_certificate()
             except AcmeClientError:
-                _LOGGER.error("ACME certification fails. Please try later.")
-                raise RemoteBackendError()
-        self._instance_domain = domain
+                await self.cloud.client.async_user_message(
+                    "cloud_remote_acme", "Home Assistant Cloud", MESSAGE_REMOTE_ACME
```

Comment:
Should we use same id, so we override the old notification if we created one successfully

Reply:
good idea